scm-dao-xml/src/main/java/sonia/scm/repository/xml/XmlRepositoryDAO.java

```diff
@@ -67,17 +76,20 @@ public XmlRepositoryDAO(PathBasedRepositoryLocationResolver repositoryLocationRe
     this.repositoryExportingCheck = repositoryExportingCheck;
 
     this.byId = new ConcurrentHashMap<>();
```

Comment:
Does this map still have to be concurrent?

Reply:
No, rewrote synchronization.